contrib/simdjson-cmake/CMakeLists.txt

```diff
@@ -1,6 +1,25 @@
-set(SIMDJSON_INCLUDE_DIR "${ClickHouse_SOURCE_DIR}/contrib/simdjson/include")
-set(SIMDJSON_SRC_DIR "${ClickHouse_SOURCE_DIR}/contrib/simdjson/src")
-set(SIMDJSON_SRC ${SIMDJSON_SRC_DIR}/simdjson.cpp)
+option (ENABLE_SIMDJSON "Use simdjson library (internal only)" ${ENABLE_LIBRARIES})
 
-add_library(simdjson ${SIMDJSON_SRC})
-target_include_directories(simdjson SYSTEM PUBLIC "${SIMDJSON_INCLUDE_DIR}" PRIVATE "${SIMDJSON_SRC_DIR}")
+if (ENABLE_SIMDJSON)
+    set (LIBRARY_DIR "${ClickHouse_SOURCE_DIR}/contrib/simdjson")
+
+    add_library (simdjson ${LIBRARY_DIR}/src/simdjson.cpp)
+
+    target_include_directories(simdjson
+        PUBLIC
```

Comment:
Use `SYSTEM PUBLIC` if you don't like to be annoyed about warnings in simdjson headers.

Reply:
@abyss7 Are you going to fix the build and merge this PR?